[2025-07-02T07:46:14.208+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-02T07:46:14.228+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-07-02T07:44:00+00:00 [queued]>
[2025-07-02T07:46:14.242+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-07-02T07:44:00+00:00 [queued]>
[2025-07-02T07:46:14.243+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-07-02T07:46:14.267+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): run_dbt_trusted> on 2025-07-02 07:44:00+00:00
[2025-07-02T07:46:14.289+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1422) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-02T07:46:14.300+0000] {standard_task_runner.py:72} INFO - Started process 1430 to run task
[2025-07-02T07:46:14.300+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'run_dbt_trusted', 'scheduled__2025-07-02T07:44:00+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmpmtavzc09']
[2025-07-02T07:46:14.305+0000] {standard_task_runner.py:105} INFO - Job 93: Subtask run_dbt_trusted
[2025-07-02T07:46:14.372+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-07-02T07:44:00+00:00 [running]> on host 24d4a3002b8f
[2025-07-02T07:46:14.452+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_trusted' AIRFLOW_CTX_EXECUTION_DATE='2025-07-02T07:44:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-02T07:44:00+00:00'
[2025-07-02T07:46:14.454+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-02T07:46:14.483+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-02T07:46:14.487+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -x; cd /opt/***/dbt && dbt run --select tag:trusted']
[2025-07-02T07:46:14.506+0000] {subprocess.py:86} INFO - Output:
[2025-07-02T07:46:14.518+0000] {subprocess.py:93} INFO - + cd /opt/***/dbt
[2025-07-02T07:46:14.520+0000] {subprocess.py:93} INFO - + dbt run --select tag:trusted
[2025-07-02T08:49:48.907+0000] {job.py:229} INFO - Heartbeat recovered after 3814.75 seconds
[2025-07-02T08:49:54.193+0000] {subprocess.py:93} INFO - [0m08:49:54  Running with dbt=1.8.0
[2025-07-02T08:49:55.022+0000] {subprocess.py:93} INFO - [0m08:49:55  Registered adapter: postgres=1.8.2
[2025-07-02T08:49:55.486+0000] {subprocess.py:93} INFO - [0m08:49:55  Found 9 models, 10 sources, 428 macros
[2025-07-02T08:49:55.494+0000] {subprocess.py:93} INFO - [0m08:49:55
[2025-07-02T08:49:55.702+0000] {subprocess.py:93} INFO - [0m08:49:55  Concurrency: 1 threads (target='dev')
[2025-07-02T08:49:55.705+0000] {subprocess.py:93} INFO - [0m08:49:55
[2025-07-02T08:49:55.721+0000] {subprocess.py:93} INFO - [0m08:49:55  1 of 4 START sql table model driven_trusted.non_pii_data ....................... [RUN]
[2025-07-02T08:49:55.837+0000] {subprocess.py:93} INFO - [0m08:49:55  1 of 4 OK created sql table model driven_trusted.non_pii_data .................. [[32mSELECT 171[0m in 0.11s]
[2025-07-02T08:49:55.840+0000] {subprocess.py:93} INFO - [0m08:49:55  2 of 4 START sql table model driven_trusted.payment_data ....................... [RUN]
[2025-07-02T08:49:55.879+0000] {subprocess.py:93} INFO - [0m08:49:55  2 of 4 OK created sql table model driven_trusted.payment_data .................. [[32mSELECT 171[0m in 0.04s]
[2025-07-02T08:49:55.884+0000] {subprocess.py:93} INFO - [0m08:49:55  3 of 4 START sql table model driven_trusted.pii_data ........................... [RUN]
[2025-07-02T08:49:55.970+0000] {subprocess.py:93} INFO - [0m08:49:55  3 of 4 OK created sql table model driven_trusted.pii_data ...................... [[32mSELECT 171[0m in 0.08s]
[2025-07-02T08:49:55.972+0000] {subprocess.py:93} INFO - [0m08:49:55  4 of 4 START sql table model driven_trusted.technical_data ..................... [RUN]
[2025-07-02T08:49:56.043+0000] {subprocess.py:93} INFO - [0m08:49:56  4 of 4 OK created sql table model driven_trusted.technical_data ................ [[32mSELECT 171[0m in 0.07s]
[2025-07-02T08:49:56.058+0000] {subprocess.py:93} INFO - [0m08:49:56
[2025-07-02T08:49:56.058+0000] {subprocess.py:93} INFO - [0m08:49:56  Finished running 4 table models in 0 hours 0 minutes and 0.56 seconds (0.56s).
[2025-07-02T08:49:56.098+0000] {subprocess.py:93} INFO - [0m08:49:56
[2025-07-02T08:49:56.099+0000] {subprocess.py:93} INFO - [0m08:49:56  [32mCompleted successfully[0m
[2025-07-02T08:49:56.100+0000] {subprocess.py:93} INFO - [0m08:49:56
[2025-07-02T08:49:56.101+0000] {subprocess.py:93} INFO - [0m08:49:56  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-07-02T08:49:57.704+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-07-02T08:49:57.801+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-02T08:49:57.802+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=run_dbt_trusted, run_id=scheduled__2025-07-02T07:44:00+00:00, execution_date=20250702T074400, start_date=20250702T074614, end_date=20250702T084957
[2025-07-02T08:49:57.873+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-02T08:49:57.927+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-02T08:49:57.931+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
